content/sensu-go/6.0/sensuctl/back-up-recover.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Back up and recover resources with sensuctl"
+linkTitle: "Back Up and Recover Resources"
+description: "Use the sensuctl command line tool to back up and recover your Sensu resources. Read this page to learn how to use sensuctl dump commands for backup and recovery."
+weight: 20
+version: "6.0"
+product: "Sensu Go"
+platformContent: false 
+menu:
+  sensu-go-6.0:
+    parent: sensuctl
+---
+
+The `sensuctl dump` command allows you to export your resources to standard out (STDOUT) or to a file.
+You can export all of your resources or a subset of them based on a list of resource types.
+The `dump` command supports exporting in `wrapped-json` and `yaml`.
+
+{{% notice note %}}
+**NOTE**: Passwords are not included when exporting users.
+You must add the `password` attribute to any exported user resources before you can use them with `sensuctl create`.
```

Comment:
We could also mention `password_hash` as an alternative to `password` (one or the other is required) link to https://docs.sensu.io/sensu-go/latest/sensuctl/#generate-a-password-hash and we need some additional text to explain how this output is used to set the password_hash for a user resource.

Reply:
I copied the note from later in the recover instructions with slight modifications.